src/test/java/org/openbrewerydb/api/BreweryApiTest.java

```diff
@@ -79,4 +84,27 @@ public void testGetBrewery() {
     assertThat(actual).isEqualTo(expected);
   }
 
+  @Test
+  public void testGetNearestBreweries() {
+    final BreweryInternal brewery = createTestBrewery(this.breweryDao);
+    GetNearestBreweriesRequest request = new GetNearestBreweriesRequest(brewery.location(), 1);
+    HttpResponse<String> response = Unirest.post(String.format("%s/search-nearest", this.apiUrl))
+        .body(JavalinJson.toJson(request))
+        .asString();
+
+    assertThat(response.getStatus()).isEqualTo(200);
+    List<Brewery> breweries = JavalinJson.fromJson(response.getBody(), GetNearestBreweriesResponse.class).breweries();
+    assertThat(breweries.size()).isEqualTo(1);
+
+    // not in radiusMeters
+    request = new GetNearestBreweriesRequest(new Location(1.0, 1.0), 1);
+    response = Unirest.post(String.format("%s/search-nearest", this.apiUrl))
+        .body(JavalinJson.toJson(request))
+        .asString();
+
+    assertThat(response.getStatus()).isEqualTo(200);
+    breweries = JavalinJson.fromJson(response.getBody(), GetNearestBreweriesResponse.class).breweries();
+    assertThat(breweries).isEmpty();
+  }
+
```

Comment:
Do you think we could have some tests where we are validating when 1/3 or 2/3 parameters are passed in and throw an appropriate error? Usually, that should be a 400